Magento translations are applied level information (#4166)
magento/devdocs: guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

## How Magento applies locales

> When the {% glossarytooltip 05099dbb-d491-4e33-a065-16035cb2d4d9 %}locale{% endglossarytooltip %} is changed for a store, Magento searches for translations in the corresponding dictionaries are applied in following manner:

Reviewer: ```suggestion
When the {% glossarytooltip 05099dbb-d491-4e33-a065-16035cb2d4d9 %}locale{% endglossarytooltip %} is changed for a store, Magento searches for and applies translations in the corresponding dictionaries:
```

1. Module translations: `<module_dir>/i18n/`
1. Theme translations: 
	1. `<parent_theme_dir>/i18n/` (iterated through all ancestor themes)
	1. `<current_theme_dir>/i18n/`
1. Translation package: `app/i18n/`

> 1. Magento database (It is good for overriding other translations whereever necessary)

Reviewer: ```suggestion
1. Magento database (translations located in this database take precedence and override translations stored in other locations.)
```

Reviewer: @abrarpathan19 thanks for clarifying. Please see my recent edit.

> 1. Magento database (It is good for overriding other translations whereever necessary)

Reviewer: Can you elaborate on when translations in the Magento database would override other translations? Is it that if translations are present in the Magento database they automatically override other translations for the same string?

Author: Hello, @erikmarr 
Yes, database translations are work on top priority,  It is a quick way for store administrators to update and translate text on the Storefront as an inline translation. 

So administrators do not need to modify CSVs. some it may happen, administrators are the non-technical user. It is a Magento good feature.

Thank you !!!

"Card Verification Number",	<translation>
```

> Translations are applied in following manner:

Reviewer: How is this order any different than what we say earlier in this topic under the 'How Magento applies locales'?

Author: Hello @eriksmarr 

Yes, I agreed My added content was duplicate,  I want to show the clear information to the user how Magento applies translation in a simple way. As per your advise I have removed duplicated content and update my point into existing content.  

Thank you.
